src/System.Runtime.Extensions/src/System.Runtime.Extensions.csproj

```diff
@@ -384,6 +384,11 @@
     <ProjectReference Include="..\..\System.Diagnostics.Debug\src\System.Diagnostics.Debug.csproj">
       <UndefineProperties>%(ProjectReference.UndefineProperties);TargetGroup</UndefineProperties>
     </ProjectReference>
+    <ProjectReference Include="..\..\System.Security.Principal\src\System.Security.Principal.csproj">
```

Comment:
I believe all of the uses of IPrincipal and PrincipalPolicy are now only in dead code, and can safely be removed